data/azure/66-azure-storage.rules

```diff
@@ -0,0 +1,28 @@
+ACTION=="add|change", SUBSYSTEM=="block", ENV{ID_VENDOR}=="Msft", ENV{ID_MODEL}=="Virtual_Disk", GOTO="azure_disk"
```

Comment:
Do we know if this files are going to get updated? Is this going to imply maintenance?